data/db.js

```diff
@@ -0,0 +1,7 @@
+import events from "../data.json";
+
+function getAllEvents() {
+  return events;
+}
+
+export { getAllEvents };
```

Comment:
It is ok, if you take a few code lines from a previous project. However, you should always adjust it, so that it matches with your project. 
```suggestion
import dishes from "../data.json";

function getAllDishes() {
  return dishes;
}

export { getAllDishes };
```